spec/model_spec.cr

```diff
@@ -9,6 +9,29 @@ private class QueryMe < LuckyRecord::Model
   end
 end
 
+private class MissingColumn < LuckyRecord::Model
+  table users do
+    field missing : String
+  end
```

Comment:
What do you think about adding a feature for `Did you mean...` for similarly named columns? That might be helpful

Reply:
Oh yeah that could be helpful